### DISCUSSION TITLE
Difference of behavior between #http and web search

### DISCUSSION THREAD

**Discussion by flefevre** [open-webui/open-webui#3032] [Q&A]
Dear all,
I am working in a reasearch laboratory. We are trying to use Open WebUI to see that capacities of generative ia for our daily tasks.:

I am able to create a prompt such "summarize the following web site #https://thewebsite

But if I activate the web search feature, if I ask "what are the latest news on generative ai?" 

It is able to call the google search api, retrieve web site of interest, but it is not able to read them
Prompt: what are the news on rolland garros?
Popup: `Something went wrong :/ HTTPSConnectionPool(host='www.rolandgarros.com', port=443): Max retries exceeded with url: /en-us/latest-news (Caused by SSLError(SSLEOFError(8, '[SSL: UNEXPECTED_EOF_WHILE_READING] EOF occurred in violation of protocol (_ssl.c:1006)')))`

If there a difference of behavior between "#https" and web search

Thanks.

François

**Comment by justinh-rahb** [comment]
the `#https://example.com` function does a straight GET request for that address and then RAGs whatever is returned before generating a response. Sometimes this fails if the website in question does not allow automated crawlers and blocks your request. The web search backends are _possibly_ smarter, they're using search APIs for one, which should only surface crawlable URLs.
